systemtest/src/test/java/io/strimzi/systemtest/ConnectClusterIT.java

```diff
@@ -145,4 +148,28 @@ public void testKafkaConnectScaleUpScaleDown() {
             assertThat(events, hasNoneOfReasons(Failed, Unhealthy, FailedSync, FailedValidation));
         }
     }
+
+    @Test
+    @JUnitGroup(name = "regression")
+    @ConnectCluster(name = CONNECT_CLUSTER_NAME, connectConfig = CONNECT_CONFIG)
+    public void testForUpdateValuesInConnectCM() {
+        List<String> connectPods = kubeClient.listResourcesByLabel("pod", "strimzi.io/type=kafka-connect");
+        replaceCm(CONNECT_CLUSTER_NAME, "healthcheck-delay", "120");
```

Comment:
Why are you doubling the original values? Any particular reason?

The problem with this is that you have just increased the healthcheck delay from 60s to 120s. And that means that you will now wait for 2 minutes for the pods to get ready instead of 1 minute. IMHO you should always just bump the values ... e.g. if the original values are `60` and `5`, use `61` and `6` unless you have some specific reason